runtime_tests/test_samv71_runtime.py

```diff
@@ -127,8 +127,8 @@ def test_samv71_ada_og(taste_project):
 
 ##
 # TASTE SAMV71 runtime supports deployment of TASTE models containing at least 8 TASTE functions (including
-# at least 4 EDS based). This is validated by the demonstration application (DemoSat2) which contains 14 EDS functions
-# and 1 Linux functions.
+# at least 4 EDS based). This is validated by the demonstration application (DemoSat2) which contains 8 EDS functions,
+# 5 C-based functions and 1 Linux function.
```

Comment:
Linux function is not deployed onto SAMV71 runtime. So it is better to write 8... + 5.... deployed onto TASTE SAMV71 Runtime.

Reply:
done